pkgs/development/arduino/arduino-core/default.nix

```diff
@@ -75,6 +105,49 @@ stdenv.mkDerivation rec {
         --replace '<BINARY_LOCATION>' "$out/bin/arduino" \
         --replace '<ICON_NAME>' "$out/share/arduino/icons/128x128/apps/arduino.png"
     ''}
+
+
+    ${stdenv.lib.optionalString (withTeensyduino == true) "
```

Comment:
This would look much nicer if it were a separate `'''...'''` block after the initial `installPhase` text block instead of inline.